internal/cmd/approve_temp/approve_temp.go

```diff
@@ -0,0 +1,32 @@
+// Copyright The gittuf Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package approve_temp
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+// New returns the cobra.Command for the `approve-temp` subcommand.
+func New() *cobra.Command {
+	var action string
+	var duration string
+
+	cmd := &cobra.Command{
+		Use:   "approve-temp",
+		Short: "Generate a temporary approval attestation",
+		Long: `Creates a signed, time-limited approval attestation for a PR or commit.
+Currently this is just a stub; full implementation will be added in later phases.`,
```

Comment:
We can omit the stub part, as we can simply place this command behind developer mode if we need to.
```suggestion
		Long: `Creates a signed, time-limited approval attestation for a PR or commit.`,
```